apps/examples/testcase/le_tc/network/Kconfig

```diff
@@ -5,6 +5,7 @@
 
 menuconfig EXAMPLES_TESTCASE_NETWORK
 	bool "Network TestCase Example"
+	depends on NET_LWIP_ERROR
```

Comment:
Is "depends on " right? not "select"?

Reply:
This is a dependency case, so it should be 'depends on' as far as I checked.